### DISCUSSION TITLE
2 q-tabs components in q-header

### DISCUSSION THREAD

**Discussion by redigaffi** [quasarframework/quasar#10724] [General - Components / Directives / etc]
Hello, I have a question about q-tabs, so, I want to have a portion of the menu aligned to the left, and another portion to the right, also the menu on the right I want to remove the "indicator-color". I have accomplished what I wanted with custom styling, but I dont think it's the correct approach.

This is my result (and also what I want)
![image](https://user-images.githubusercontent.com/821517/133880497-84244868-5b81-4f48-ae83-841c5da899a7.png)

Here is the code: https://pastebin.com/H3eVutEp

Login component is a  q-tab in it's own component
`
<q-tab align="right" indicator-color="blue" :label="logInButtonText" icon="fas fa-user" @click="login" />
`

Appreciate any ideas

**Comment by ontwikkelfabriek** [comment]
@redigaffi 

I would put all the tabs( all components inside the header)  inside the toolbar . Then you can use flexbox and `q-spacer` to align items to left and right. Like these examples:
https://quasar.dev/layout/header-and-footer#example--playing-with-qtabs
https://quasar.dev/vue-components/toolbar#example--with-tabs

**Reply by redigaffi** [reply]
Hello, thanks for the fast reply. Actually adding a QSpace element solves the issue in having 2 q-tabs.

But there's a small thing still missing, I want to remove the indicator-color for the menu elements on the right, I can only accomplish that by having 2 q-tabs. 

New code: https://pastebin.com/aMmpsbQ2

Any idea?

Thank you!

**Comment by ontwikkelfabriek** [comment]
@redigaffi 

>  I want to remove the indicator-color for the menu elements on the right, I can only accomplish that by having 2 q-tabs.

Instead of posting a codelisting in pastbin please create a `codepen.io` so we can provide better support. You can easily extend an existing example codepen.io in the Quasar doc.  Or use this one ( Qv2):
https://codepen.io/rstoenescu/pen/xxEvBLR
